pkgs/servers/uwsgi/default.nix

```diff
@@ -91,12 +92,15 @@ stdenv.mkDerivation rec {
     inherit python2 python3;
   };
 
+  php8 = builtins.head (builtins.splitVersion php.version) == "8";
+  php8_no_version = ''sed -e "s/ + php_version//" -i plugins/php/uwsgiplugin.py'';
+
   postPatch = ''
     for f in uwsgiconfig.py plugins/*/uwsgiplugin.py; do
       substituteInPlace "$f" \
         --replace pkg-config "$PKG_CONFIG"
     done
-  '';
+  '' + (lib.optionalString php8 php8_no_version);
```

Comment:
While it doesn't really change anything function-wise, this is kind of awkward to read, IMHO.

Why not just have this whole block in the `postPatch` as:
```
${lib.optionalString (lib.versionAtLeast php80.version "8") ''
  sed -e "s/ + php_version//" -i plugins/php/uwsgiplugin.py'';
''}
```
or 
```
+ (lib.optionalString (lib.versionAtLeast php80.version "8") ''
  sed -e "s/ + php_version//" -i plugins/php/uwsgiplugin.py'';
'')
```

Reply:
yeah, sure. I still lack the skills to write idiomatic nix. let me fix that

Reply:
changed it, does it look ok now?